geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl

```diff
@@ -4,6 +4,262 @@ This file includes migration steps for each release of c2cgeoportal.
 Version 2.5.0
 =============
 
+Information to know before starting the upgrade
```

Comment:
```suggestion
Informations to know before starting the upgrade
```

Reply:
As I know Information has never 's', not?

Reply:
Seems you're right on this one (as says google translate).